icubam/db/test_synchronizer.py

```diff
@@ -236,3 +236,11 @@ def test_export_icus(self):
       str_buf.splitlines(),
       len(icus) + 1, "Number of rows should be the same."
     )
+
+  def test_import_bedcounts(self):
+    with open("resources/test/icu2.csv") as csv_f:
+      self.csv.sync_icus_from_csv(csv_f, False)
+    with open("resources/test/bedcounts.csv") as csv_f:
```

Comment:
In the long term I we should change the folder where these files are stored https://github.com/icubam/icubam/issues/296#issuecomment-616095233